src/Stratis.SmartContracts.CLR.Validation/Stratis.SmartContracts.CLR.Validation.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <Version>0.0.3-beta</Version>
+    <Version>1.0.2.0-beta</Version>
```

Comment:
Seems to be `1.0.2` elsewhere, is `1.0.2.0` deliberate?

Reply:
Hmm, seems to be an even split between the two actually, some use a.b.c.d and others just a.b.c